lib/v3-lab.js

```diff
@@ -93,7 +92,7 @@ function V3DocumentMixin(documentClass) {
 window.MathJax = {
     loader: {
         load: [
-            'input/tex',
+            'input/tex-full',
```

Comment:
See my previous comment in PR #226 on `tex-full` vs `tex`.

Reply:
The only comment there is the one about `autobold`.  Is that the one you are talking about?  I'm not sure it affects the lab, since its the lab expression is not in a bold setting.  

Is there maybe some comments that you haven't submitted?

Reply:
That's odd, I definitely made it. After submitting the review though and now it seems to have vanished.
Anyway the comment was: Since we have `tex-base` why do we need to call it `tex-full` instead of simply `tex`?